### DISCUSSION TITLE
Аdd 2d simple sprite editor.

### DISCUSSION THREAD

**Discussion by Del37** [godotengine/godot-proposals#6487] [Editor]
It would be very cool to add just a 2D picture editor so that you could at least just repaint body parts, add some different sized pisels and other basic things.

Inspector -> sprite -> Texture -> open editor.

**Comment by AThousandShips** [comment]
This seems like a lot of work to implement a feature that would only be used in very specific conditions, which has to be maintained.

When a feature has a very limited use it becomes very hard to maintain, few people will use it so few people will notice bugs, it also adds to the size of the engine, feature bloat is to be avoided.

In general assets are intended to be provided to the engine from other sources, not created in the engine, adding our own very primitive editor would be reinventing the wheel.

**Reply by AThousandShips** [reply]
Most operating systems come with a built in editor which would be more capable than what a simple editor in Godot anyway

**Reply by Zireael07** [reply]
There ARE a couple of simple 2D sprite editors done in Godot anyway... OP should search assets store and Github
